bench/app.py

```diff
@@ -93,58 +95,64 @@ def remove_from_excluded_apps_txt(app, bench_path='.'):
 		return write_excluded_apps_txt(apps, bench_path=bench_path)
 
 def get_app(git_url, branch=None, bench_path='.', skip_assets=False, verbose=False,
-	postprocess = True):
+		postprocess=True, overwrite=False):
 	# from bench.utils import check_url
 	try:
 		from urlparse import urljoin
 	except ImportError:
 		from urllib.parse import urljoin
 
-	if not check_url(git_url, raise_err = False):
+	if not check_url(git_url, raise_err=False):
 		orgs = ['frappe', 'erpnext']
 		for org in orgs:
-			url = 'https://api.github.com/repos/{org}/{app}'.format(org = org, app = git_url)
+			url = 'https://api.github.com/repos/{org}/{app}'.format(org=org, app=git_url)
 			res = requests.get(url)
 			if res.ok:
-				data    = res.json()
+				data = res.json()
 				if 'name' in data:
 					if git_url == data['name']:
-						git_url = 'https://github.com/{org}/{app}'.format(org = org, app = git_url)
+						git_url = 'https://github.com/{org}/{app}'.format(org=org, app=git_url)
 						break
 
-	#Gets repo name from URL
+	# Gets repo name from URL
 	repo_name = git_url.rsplit('/', 1)[1].rsplit('.', 1)[0]
-	logger.info('getting app {}'.format(repo_name))
 	shallow_clone = '--depth 1' if check_git_for_shallow_clone() else ''
 	branch = '--branch {branch}'.format(branch=branch) if branch else ''
 
-	exec_cmd("git clone -q {git_url} {branch} {shallow_clone} --origin upstream".format(
-				git_url=git_url,
-				shallow_clone=shallow_clone,
-				branch=branch),
-			cwd=os.path.join(bench_path, 'apps'))
-
-	#Retrieves app name from setup.py
+	if os.path.isdir(os.path.join(bench_path, 'apps', repo_name)):
+		# application directory already exists
+		# prompt user to overwrite it
+		if overwrite or click.confirm('''A directory for the application "{0}" already exists.
+Do you want to continue and overwrite it?'''.format(repo_name)):
+			shutil.rmtree(os.path.join(bench_path, 'apps', repo_name))
+		elif click.confirm('''Do you want to reinstall the existing application?''', abort=True):
+			app_name = get_app_name(bench_path, repo_name)
+			print("Reinstalling {0}".format(app_name))
+			install_app(app=app_name, bench_path=bench_path, verbose=verbose, skip_assets=skip_assets)
+			sys.exit()
+
+	logger.info('Getting app {0}'.format(repo_name))
+	exec_cmd("git clone {git_url} {branch} {shallow_clone} --origin upstream".format(
+		git_url=git_url,
+		shallow_clone=shallow_clone,
+		branch=branch),
+		cwd=os.path.join(bench_path, 'apps'))
+
+	app_name = get_app_name(bench_path, repo_name)
+	print("Installing {0}".format(app_name))
```

Comment:
![Screenshot 2020-02-25 at 1 06 17 PM](https://user-images.githubusercontent.com/36654812/75224975-a1a4cb80-57cf-11ea-8eb1-f8f80d3bd97e.png)

Is that print statement really necessary? `(along with bench logger's outputs)`